dashboard/app/views/layouts/application.html.haml

```diff
@@ -1,6 +1,13 @@
 !!! 5
 %html{dir: locale_dir}
   %head
+    // The Crowdin In-Context tool (https://crowdin.com/project/codeorg/settings#in-context) allows us to view original & translated strings in Crowdin.
+    // "in-TL" is a pseudo-language locale where the Crowdin strings are stored
+    - if request.locale === "in-TL"
+      :javascript
+        var _jipt = [];
```

Comment:
Just for my curiosity, what does `jipt` stand for?

Reply:
Just In Place Translations